src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -12,40 +12,44 @@ redirects:
-   * **Redirect is Failure (for ping).** If a redirect result occurs when Redirect is Failure is enabled, Synthetics categorizes it as a failure (rather than following the redirect and checking the resulting URL).
-3. Select the locations where you want your monitor to run.
-4. Choose a frequency to determine how often each location will run your monitor.
-5. Optional: Set up alert notifications.
-6. Select **Create my monitor** to confirm.
-7. Wait a few minutes; then from the [Monitors index](#bookmark) check your monitor.
 
-## 2. View all monitors with the Monitors index page [#bookmark]
+   * **Bypass HEAD request (for ping monitors).** This option skips the default HEAD request and instead uses the GET verb with a ping check. GET requests will always happen if the HEAD request fails.
+   * **Redirect is Failure (for ping).** If a redirect result occurs when `Redirect is Failure` is enabled, your synthetic monitors categorize it as a failure (rather than following the redirect and checking the resulting URL).
+
+4. Select the locations where you want your monitor to run. It is recommended to select at least three locations to avoid false positives. In other words, if at least one location returns a successful result then the endpoint must be up and triggering of an alert can be avoided.
+5. Depending on the monitor type, choose to either Save monitor, Validate, or Write script.
+6. You'll be taken to the [Summary page](#summary-page) where results will start to appear as they are received.
+
+## 2. Get a quick summary of how a monitor is running [#summary-page]
+
+TBD, screenshots, what's useful on this page?
+
+Once on the Summary page, you'll be presented with the most relevant information to assist you in determining the status of this monitor. If an alert has triggered and created an active incident, click to open the critical alert in a new tab for this monitor, or click to manage alert policies for all monitors at the top of the page.
+![](/src/images/synthetics-monitor-critical-alert.png)
+
+## 3. View all monitors on the Synthetics Monitors page [#synthetics-monitors]
 
-Continuous application performance monitoring is essential to ensure that web services are in place, working correctly, and error-free. Synthetic monitoring provides this type of assurance by performing automated tests on your web application for each selected location—noting downtime instances (“violations”) and collecting aggregate numbers, results, and detailed statistics for each page resource.
+Continuous application performance monitoring is essential to ensure that web services are in place, working correctly, and error-free. Synthetic monitoring provides this type of assurance by performing automated tests on your web application for each selected location—noting downtime instances ("violations") and collecting aggregate numbers, results, and detailed statistics for each page resource.
```

Comment:
@keegoid-nr Location as in what's being monitored (endpoints, applications?) or locations as in where the IP is pinging from, or locations as in something else?

Reply:
can you describe what you mean by 'page resource', or what a page resource provides from a customer pOV?